.travis.yml

```diff
@@ -60,3 +60,31 @@ matrix:
         - cd iOS/Sample
         - xcodebuild -showsdks
         - xcodebuild clean build -workspace Sample.xcworkspace -scheme Pods-Sample -sdk iphonesimulator11.4
+
+    - language: android
```

Comment:
Awesome! As a next step (not necessarily part of this), could we run this on Linux hosts to speed things up. There's actually support for [multi OS testing](https://docs.travis-ci.com/user/multi-os/) that I didn't know about and we could simply skip Android builds on MacOS and vice versa.

Reply:
I run android builds on linux.